ocs_ci/helpers/helpers.py

```diff
@@ -4480,3 +4487,103 @@ def verify_log_exist_in_pod_logs(
     )
     logger.info(f"logs osd:{pod_logs}")
     return expected_log in pod_logs
+
+
+def retrieve_cli_binary(cli_type="mcg"):
+    """
+    Download the MCG-CLI/ODF-CLI binary and store it locally.
+
+    Args:
+        cli_type (str): choose which bin file you want to download ["odf" -> odf-cli , "mcg" -> mcg-cli]
+
+    Raises:
+        AssertionError: In the case the CLI binary is not executable.
+
+    """
+    semantic_version = version.get_semantic_ocs_version_from_config()
+    if cli_type == "odf" and semantic_version < version.VERSION_4_15:
+        raise Exception(f"odf cli tool not supported on ODF {semantic_version}")
```

Comment:
I would rather define some NotSupportedException in https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/exceptions.py for it than using generic Exception.

Reply:
done